services/net/transcription/TranscriptionManager.cs

```diff
@@ -239,11 +241,19 @@ private async Task<string> RequestTranscriptionAsync(byte[] data, string languag
             if (result.Reason == ResultReason.RecognizedSpeech)
             {
                 sb.Append(result.Text);
+                this.Logger.LogDebug("Speech transcription process \"{text}...\"", result.Text?[0..Math.Min(result.Text.Length, 25)]);
             }
+            // TODO: Handle other reasons.
```

Comment:
There are some other interesting reasons like `Intent` that could be valuable to capture.